svc-bip-api/src/integrationTest/java/gov/va/vro/bip/RabbitMqIntegrationTest.java

```diff
@@ -4,14 +4,7 @@
 
 import com.fasterxml.jackson.databind.JsonNode;
 import com.fasterxml.jackson.databind.ObjectMapper;
-import gov.va.vro.bip.model.BipClaimResp;
-import gov.va.vro.bip.model.BipContentionResp;
-import gov.va.vro.bip.model.BipUpdateClaimResp;
-import gov.va.vro.bip.model.ClaimStatus;
-import gov.va.vro.bip.model.RequestForUpdateClaimStatus;
-import gov.va.vro.bip.model.UpdateContention;
-import gov.va.vro.bip.model.UpdateContentionModel;
-import gov.va.vro.bip.model.UpdateContentionReq;
+import gov.va.vro.bip.model.*;
```

Comment:
Do we want to use wildcard imports?

Reply:
wildcard import is now cleaned up in the latest squash merge